brickpi3/src/ext/Cron.h

```diff
@@ -0,0 +1,121 @@
+#include <chrono>
+#include <string>
+#include <sstream>
+#include <vector>
+#include <iterator>
+
+namespace Bosma {
```

Comment:
And this looks like we are now duplicating the clones of Bosma.  Not looking good :) We should start by checking the provenance of this Bosma code.

Reply:
The only way to be able to avoid duplication is to create a package that hosts these headers and then have both example packages depend on *that* package.

That seemed like it would be a bit too much.

So I opted to just place a copy of these headers in each of the example packages.

It's just example code, so I don't believe it's going to be too much pain keeping those copies around.